package.json

```diff
@@ -44,16 +44,16 @@
     "homepage": "https://github.com/TinkoffCreditSystems/ng-polymorpheus#README",
     "schematics": "./schematics/collection.json",
     "dependencies": {
-        "@angular/animations": "9.1.13",
-        "@angular/common": "9.1.13",
-        "@angular/compiler": "9.1.13",
-        "@angular/core": "9.1.13",
-        "@angular/forms": "9.1.13",
-        "@angular/platform-browser": "9.1.13",
-        "@angular/platform-browser-dynamic": "9.1.13",
-        "@angular/platform-server": "9.1.13",
-        "@angular/router": "9.1.13",
-        "@nguniversal/express-engine": "^9.1.1",
+        "@angular/animations": "12.0.0",
```

Comment:
why don't we use "^12.0.0"?

Reply:
That's how ng update did it. I think it doesn't matter much :)